test/IRGen/package_resilience.swift

```diff
@@ -214,29 +226,43 @@ package class ClassWithResilientThenEmpty {
 
 // CHECK-LABEL: define{{.*}} swiftcc {{i32|i64}} @"$s18package_resilience38memoryLayoutDotSizeWithResilientStructSiyF"()
 public func memoryLayoutDotSizeWithResilientStruct() -> Int {
-  // CHECK: ret [[INT]] [[#WORDSIZE + WORDSIZE]]
+// CHECK: [[FIELD_VALUE:%.*]] = load ptr, ptr {{.*}},
+// CHECK-NEXT: [[FIELD_PTR:%.*]] = getelementptr inbounds %swift.vwtable, ptr [[FIELD_VALUE]],
+// CHECK-NEXT: [[SIZE:%.*]] = load [[INT]], ptr [[FIELD_PTR]],
+// CHECK-NEXT: ret [[INT]] [[SIZE]]
   return MemoryLayout<Size>.size
 }
 
 // CHECK-LABEL: define{{.*}} swiftcc {{i32|i64}} @"$s18package_resilience40memoryLayoutDotStrideWithResilientStructSiyF"()
 public func memoryLayoutDotStrideWithResilientStruct() -> Int {
-  // CHECK: ret [[INT]] [[#WORDSIZE + WORDSIZE]]
+// CHECK: [[FIELD_VALUE:%.*]] = load ptr, ptr {{.*}},
+// CHECK-NEXT: [[FIELD_PTR:%.*]] = getelementptr inbounds %swift.vwtable, ptr [[FIELD_VALUE]],
+// CHECK-NEXT: [[SIZE:%.*]] = load [[INT]], ptr [[FIELD_PTR]],
+// CHECK-NEXT: ret [[INT]] [[SIZE]]
   return MemoryLayout<Size>.size
```

Comment:
It looks like there was a copy paste mishap here in the original code. I would expect this test to call `.stride`. As is the test is identical to the previous use case.

Reply:
Yes, this should be stride.